controllers/calendar/helpers/RestDefinitions.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\calendar\helpers;
+
+use humhub\modules\calendar\models\CalendarEntry;
+use humhub\modules\calendar\models\CalendarEntryParticipant;
+use humhub\modules\rest\definitions\ContentDefinitions;
+use humhub\modules\rest\definitions\UserDefinitions;
+
+class RestDefinitions
+{
+    public static function getCalendarEntry(CalendarEntry $entry)
+    {
+        return [
+            'id' => $entry->id,
+            'title' => $entry->title,
+            'description' => $entry->description,
+            'start_datetime' => $entry->start_datetime,
+            'end_datetime' => $entry->end_datetime,
+            'all_day' => (int)$entry->all_day,
+            'participation_mode' => (int)$entry->participation_mode,
+            'color' => $entry->color,
+            'allow_decline' => (int)$entry->allow_decline,
+            'allow_maybe' => (int)$entry->allow_maybe,
+            'time_zone' => $entry->time_zone,
+            'location' => $entry->location ?? "",
```

Comment:
this is the only new thing?

Reply:
Hi @donni106 
Yes, this is the only key we added. And '??' means if there is no value in $entry->location and show blank. it is similar to the ternary operator or if-else condition.

Reply:
i know the operator, yes. but why have you decided to use it compared with all the other keys which are not falling back to something?

Reply:
Okay, then we will remove this operator.